presto-product-tests/src/main/java/com/facebook/presto/tests/CreateDropViewTests.java

```diff
@@ -100,4 +107,24 @@ public void dropViewTest()
                     .failsWithMessage("does not exist");
         });
     }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testViewOwnerPrivileges()
+    {
+        String viewName = "alice_view";
+        String tableName = "alice_table";
+        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
+
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        executeWith(createViewAs(viewName, format("SELECT * FROM %s", tableName), aliceExecutor), view -> {
+            assertThat(aliceExecutor.executeQuery(format("SHOW GRANTS ON %s", viewName)))
+                    .containsOnly(ImmutableList.of(
+                            row("alice", "hive", "default", viewName, "SELECT", Boolean.TRUE),
```

Comment:
That's a valid point. Though we have to keep in mind the implications of testing via Hive. Testing this with Hive permissions requires us to configure Hive with SQL Standard Authorization enabled which I have been unsuccessful at. And we will also have to add this functionality in Tempto. We should probably not target doing this for this release then.